Tests/DangerTests/DangerDSLResources/DangerDSLGitHub.swift

```diff
@@ -0,0 +1,2126 @@
+                "payload":
+                  "tree a30d9d8be16847c33eb50483a653f27475f197a4\\nparent 4cf1e41f72516a4135f1738c47f7dd3d421ff3c4\\nauthor Ash Furrow <ash@ashfurrow.com> 1469567860 -0400\\ncommitter Ash Furrow <ash@ashfurrow.com> 1469567860 -0400\\n\\n[Deps] Updates dependencies for Swift 2.2.\\n"
+              }
+            },
+            "url": "https://api.github.com/repos/artsy/eidolon/commits/d0d72ec5b5ee90c2513a8aafb48911ae5bcdf4ac",
+            "html_url": "https://github.com/artsy/eidolon/commit/d0d72ec5b5ee90c2513a8aafb48911ae5bcdf4ac",
+            "comments_url":
+              "https://api.github.com/repos/artsy/eidolon/commits/d0d72ec5b5ee90c2513a8aafb48911ae5bcdf4ac/comments",
+            "author": {
+              "login": "ashfurrow",
+              "id": 498212,
+              "avatar_url": "https://avatars3.githubusercontent.com/u/498212?v=4",
+              "gravatar_id": "",
+              "url": "https://api.github.com/users/ashfurrow",
+              "html_url": "https://github.com/ashfurrow",
+              "followers_url": "https://api.github.com/users/ashfurrow/followers",
+              "following_url": "https://api.github.com/users/ashfurrow/following{/other_user}",
+              "gists_url": "https://api.github.com/users/ashfurrow/gists{/gist_id}",
+              "starred_url": "https://api.github.com/users/ashfurrow/starred{/owner}{/repo}",
+              "subscriptions_url": "https://api.github.com/users/ashfurrow/subscriptions",
+              "organizations_url": "https://api.github.com/users/ashfurrow/orgs",
+              "repos_url": "https://api.github.com/users/ashfurrow/repos",
+              "events_url": "https://api.github.com/users/ashfurrow/events{/privacy}",
+              "received_events_url": "https://api.github.com/users/ashfurrow/received_events",
+              "type": "User",
+              "site_admin": false
+            },
+            "committer": {
+              "login": "ashfurrow",
+              "id": 498212,
+              "avatar_url": "https://avatars3.githubusercontent.com/u/498212?v=4",
+              "gravatar_id": "",
+              "url": "https://api.github.com/users/ashfurrow",
+              "html_url": "https://github.com/ashfurrow",
+              "followers_url": "https://api.github.com/users/ashfurrow/followers",
+              "following_url": "https://api.github.com/users/ashfurrow/following{/other_user}",
+              "gists_url": "https://api.github.com/users/ashfurrow/gists{/gist_id}",
+              "starred_url": "https://api.github.com/users/ashfurrow/starred{/owner}{/repo}",
+              "subscriptions_url": "https://api.github.com/users/ashfurrow/subscriptions",
+         
```

Comment:
^ - this might have got revoked (check your emails) - unless it is one of mine?

Reply:
Is the one on the fixtures, but anyway is just to test it parses it correctly, then shouldn't matter